### ISSUE TITLE
Mpii precision

### ISSUE THREAD

**Issue by xiaochenwise** [Jeff-sjtu/res-loglikelihood-regression#32]
Simplepose's accuracy on mPII validation set is 33.911 in his paper, why is it 25.4 in yours? The MPII annotations used in your code also appear to be different from other networks, such as SimpleBaseline.
![image](https://user-images.githubusercontent.com/71183069/154227756-3f2f4962-c991-4433-a480-453cab7b6f88.png)

**Comment by jeffffffli** [comment]
Hi @xiaochenwise, [the paper](https://openaccess.thecvf.com/content_ECCV_2018/papers/Bin_Xiao_Simple_Baselines_for_ECCV_2018_paper.pdf) of simplepose did not provide results on MPII.

**Comment by GloryyrolG** [comment]
Hi there @xiaochenwise @Jeff-sjtu , I guess there might be a disagreement about the PCKh@0.1 evaluation between two repos (this RLE/[MMPose](https://download.openmmlab.com/mmpose/top_down/deeppose/deeppose_res50_mpii_256x256_rle_20220504.log.json) vs. SimpleBaseLine). I reevaluated with [SBL's script](https://github.com/leoxiaobin/deep-high-resolution-net.pytorch) and got ~33.
